Google.Cloud.Functions.Invoker/EntryPoint.cs

```diff
@@ -81,7 +91,7 @@ private static RequestDelegate BuildHandler(Assembly functionAssembly, string[]
 
         private static int DeterminePort(string[] args)
         {
-            var environmentVariable = Environment.GetEnvironmentVariable("PORT");
+            var environmentVariable = Environment.GetEnvironmentVariable(PortEnvironmentVariable);
             if (!string.IsNullOrEmpty(environmentVariable))
             {
                 if (!int.TryParse(environmentVariable, NumberStyles.None, CultureInfo.InvariantCulture, out var parsed))

```

Comment:
nit: on the exception message, two lines below part => parse.

Reply:
Thanks - fixed, and modified to refer to the constant.